client/state/sites/selectors.js

```diff
@@ -148,29 +147,22 @@ export function isJetpackSite( state, siteId ) {
 }
 
 /**
- * Returns true if site is a Pressable Jetpack site or site during a transfer, false if the site is hosted on
- * WordPress.com or is a regular Jetpack, or null if the site is unknown.
+ * Returns true if site is a Automated Transfer site, false if not and null if unknown
```

Comment:
Please add new selectors to the [global selector dir](https://github.com/Automattic/wp-calypso/tree/master/client/state/selectors). This allows us to search for them in dev docs.